src/main/java/org/opentripplanner/util/TranslatedString.java

```diff
@@ -75,7 +81,7 @@ public static I18NString getI18NString(Map<String, String> translations) {
      * @return The available translations
      */
     public List<Entry<String,String>> getTranslations() {
-        return new ArrayList<Entry<String, String>>(translations.entrySet());
+        return new ArrayList<>(translations.entrySet());
```

Comment:
```suggestion
        return List.of(translations.entrySet());
```

Reply:
`List.of()` only works with arrays / varargs, so the `Set` would need to be converted. I'm not sure this is cleaner:
```
        return List.of(translations.entrySet().toArray((IntFunction<Entry<String, String>[]>) Entry[]::new));
```